library/common/http/client.h

```diff
@@ -228,9 +228,7 @@ class Client : public Logger::Loggable<Logger::Id::http> {
     void readDisable(bool disable) override;
     uint32_t bufferLimit() override { return 65000; }
     // Not applicable
-    void setAccount(Buffer::BufferMemoryAccountSharedPtr) override {
-      PANIC("buffer accounts unsupported");
```

Comment:
what's the reason/thoughts behind this?

(this is nothing critical, but just want to know)

Reply:
👍